src/components/branding-strategy.html

```diff
@@ -0,0 +1,46 @@
+<div class="lg:max-w-7xl mx-auto md:w-3/4 w-11/12">
+    <div class="pt-28 pb-10 flex justify-center flex-col">
+        <p class="lg:text-2xl text-lg">
+            We are a Berlin-based communications agency specialising in helping
+            companies and brands find their voice and position themselves. Whether
+            it's brand building, identity sharpening, or communication strategy, we
+            bring a fresh perspective to established structures, develop tailor-made
+            communication solutions, and ensure that messages resonate. With over a
+            decade of experience, we have honed our skills in giving brands their
+            unmistakable identity and making them visible in the long term.
+        </p>
+        <div class="sticky top-0">
+            <div class="flex justify-center z-10">
+                <img src="src/assets/Vector 6.png" class="mt-10 w-36" alt="Floating Vector" />
+            </div>
+        </div>
+    </div>
+    <div class="flex flex-wrap py-28 space-y-10 md:space-y-0">
+        <div class="lg:w-1/2 w-full">
+            <ul class="lg:text-3xl text-xl font-bold w-3/4 xl:space-y-10 space-y-4 mx-auto">
+                <li>Brand strategy</li>
+
+                <li>Corporate Identity & Website development</li>
+
+                <li>SEO & copywriting</li>
+
+                <li>Media Relations</li>
+
+                <li>Digital marketing & content production</li>
+
+                <li>Influencer relations</li>
+            </ul>
```

Comment:
You can remove that spaces, it seems like the code is too much while it is really not!
```suggestion
            <ul class="lg:text-3xl text-xl font-bold w-3/4 xl:space-y-10 space-y-4 mx-auto">
                <li>Brand strategy</li>
                <li>Corporate Identity & Website development</li>
                <li>SEO & copywriting</li>
                <li>Media Relations</li>
                <li>Digital marketing & content production</li>
                <li>Influencer relations</li>
            </ul>
```